tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -64,9 +64,8 @@ class SbtServerSuite
   test("generate") {
     def sbtLaunchJar = workspace.resolve(".bsp/sbt-launch.jar")
     def sbtBspConfig = workspace.resolve(".bsp/sbt.json")
-    def isBspConfigValid = sbtBspConfig.readText.contains(
-      s"--sbt-launch-jar=${sbtLaunchJar.toString()}"
-    )
```

Comment:
So I'm not 100% sure what exactly we were meant to be testing here or if it was just a sanity check to ensure it has the launch jar listed @adpi2?

I just switching this around a bit to only check. to see if they launch jar is in there, and no longer look for the `--sbt-launch-jar` since that now looks to be `-Dsbt.script=` in 1.6.x and no longer actually points to the launch jar in that location.